frontend/src/metabase/questions/containers/CollectionEditorForm.jsx

```diff
@@ -31,23 +31,27 @@ import { normal } from "metabase/lib/colors";
         // pick a random color to start so everything isn't blue all the time
         color: normal[Math.floor(Math.random() * normal.length)]
     }
-})
-export default class CollectionEditorForm extends Component {
+}
+
+export const CollectionEditorFormActions = ({ handleSubmit, invalid, onClose, fields}) =>
+    <div>
+        <Button className="mr1" onClick={onClose}>
+            Cancel
+        </Button>
+        <Button primary disabled={invalid} onClick={handleSubmit}>
+            { fields.id.value != null ? "Update" : "Create" }
+        </Button>
+    </div>
+
+export class CollectionEditorForm extends Component {
```

Comment:
see https://flow.org/en/docs/frameworks/react/#toc-adding-types-for-react-component-props for the most modern way to add Flow prop types